content/guides/models/app/features/panels/line-plot/reference.md

```diff
@@ -62,7 +62,7 @@ You can aggregate all of the runs by turning on grouping, or group over an indiv
 
 ## Smoothing
 
-You can set the [smoothing coefficient]({{< relref "/support/formula_smoothing_algorithm.md" >}}) to be between 0 and 1 where 0 is no smoothing and 1 is maximum smoothing.
+You can set the [smoothing coefficient]({{< relref "formula_smoothing_algorithm.md" >}}) to be between 0 and 1 where 0 is no smoothing and 1 is maximum smoothing.
```

Comment:
Silly change to retrigger Cloudflare (maybe)
```suggestion
You can set the [smoothing coefficient]({{< relref "formula_smoothing_algorithm.md" >}}) to be between 0 and 1 where 0 is no smoothing and 1 is maximum smoothing.

```

Reply:
if this is just to retrigger CI, in the future you can make an empty commit, too:

```
git commit --allow-empty -m "empty commit to trigger build"
```

i use this all the time when i'm testing new github actions integrations in my repos. :slightly_smiling_face:

Reply:
Good tip. Nevertheless, it didn't work. Feel free to pop my commit off, and then perhaps if you rebase on `main` (low risk since it hasn't had review yet), maybe Cloudflare will wake up.